main.js

```diff
@@ -668,6 +742,10 @@ function delObjects(objs, callback) {
  * @return none
  */
 function checkHost(callback) {
+    if (compactGroupController) {
```

Comment:
?!?
Comment pls

Reply:
added: only main host controller needs to check/fix the host assignments from the instances